examples/resources/snowflake_grant_database_role/import.sh

```diff
@@ -0,0 +1,2 @@
+# format is database_role_name (string) | object_type (ROLE|DATABASE ROLE|SHARE) | grantee_name (string)
```

Comment:
Hmm, the docs were not generated for this new resource (the check did not fail btw, we should check why).

Reply:
docs generated